app/controllers/api/v2/reservations_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Api::V2::ReservationsController < ApplicationController
+  before_action :authorize, only: %i[create update destroy]
+  before_action :set_reservation, only: %i[update destroy]
+
+  def index
+    # @reservations = @user.reservations.all
+    # render json: @reservations
+
```

Comment:
- [ ] [OPTIONAL] Kindly delete the comments line in code to make your file professional